scripts/prepare-for-release

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# Script to:
+## 1) create and checkout a new branch with the latest tag name
+## 2) update AEMM and Helm chart versions
+## 3) commit release prep changes to new branch
+## 4) create a PR from the new branch to upstream/master
+
+## The following files are updated in step (2):
+### - helm charts' Chart.yaml
+### - helm charts' values.yaml
+### - helm charts' README
+### - README.md
+
+# Prerequisites:
+## 1) git and gh cli (make sure to use gh cli before running this script as authentication is required)
```

Comment:
soo not driven by GitHub Token?

Reply:
thats correct. 
When using gh cli for the first time, when a gh command is run, it directs you to the browser to allow access / authenticate.